tidb-cloud/integrate-tidbcloud-with-n8n.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Integrate TiDB Cloud with n8n
+summary: Learn the use of TiDB Cloud node in n8n.
+---
+
+# Integrate TiDB Cloud with n8n
+
+[n8n](https://n8n.io/) is an extendable workflow automation tool. With a [fair-code](https://faircode.io/) distribution model, n8n will always have visible source code, be available to self-host, and allow you to add your custom functions, logic, and apps. 
+
+This document introduces how to build an auto-workflow: create a TiDB Cloud Serverless Tier cluster, gather Hack News RSS, store it to TiDB and send a briefing email.
```

Comment:
![image](https://user-images.githubusercontent.com/3690895/208296229-9842b2fe-9ce5-4a71-bd49-f817f068b49e.png)
Hacker News or Hack News?

Reply:
@zhangyangyu Thank you very much. I will correct it.